data/mods/gen5/random-teams.ts

```diff
@@ -390,7 +392,7 @@ export class RandomGen5Teams extends RandomGen6Teams {
 			return 'Lustrous Orb';
 		}
 		if (counter.damagingMoves.length >= 4 && ability !== 'Sturdy') {
-			return (species.name === 'Deoxys-Attack' || counter.Normal || !!counter.priority) ? 'Life Orb' : 'Expert Belt';
+			return (hasMove['uturn']) ? 'Expert Belt' : 'Life Orb';
```

Comment:
```suggestion
			return hasMove['uturn'] ? 'Expert Belt' : 'Life Orb';
```

Reply:
Is this just for the Infernape thing, or do U-turners generally want to avoid LO (because of hazard damage ig)?